### PR TITLE
Vratjanits

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

File: src/mender/bootstrap/bootstrap.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ def now(
         log.info("Device bootstrapped successfully")
         return private_key
     except FileNotFoundError:
-        log.error(f"The directory in the path: {private_key_path} not found")
+        log.error("The directory in the path: %s not found", private_key_path)
```
Comment (lluiscampos) [inline src/mender/bootstrap/bootstrap.py:48]: Curious: why this last commit? We have been adopting f-strings across repos now (all the recent code) so if there is a good reason we should stop this practice in the rest of the repos too.

File: src/mender/_version.py [hunk 2]
```diff
@@ -11,4 +11,4 @@
 #    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
-__version__ = "1.0.0"
+__version__ = "master"
```
Comment (lluiscampos) [inline src/mender/_version.py:14]: Shouldn't this be `1.1.dev1` or something like that? Then I can upgrade from 1.0.0 to this, and eventually from this to 1.1.0

**Review by lluiscampos — APPROVED** [review]

File: src/mender/bootstrap/bootstrap.py (hunk 1 above)
Comment (oleorhagen) [reply src/mender/bootstrap/bootstrap.py:48]: This is minor, and not really important, but it's in response to this:
https://vald-phoenix.github.io/pylint-errors/plerr/errors/logging/W1202.html

File: src/mender/_version.py (hunk 2 above)
Comment (oleorhagen) [reply src/mender/_version.py:14]: Hmm, ur probably right. I just need it not to be `1.0.0`. 

Thinking about this a little :thinking: I guess `master` is not preposterous, as it really is `master`. So maybe master here, and a separate `1.1.dev` branch? Or? 

I'm all open for suggestions discussion here

File: src/mender/bootstrap/bootstrap.py (hunk 1 above)
Comment (oleorhagen) [reply src/mender/bootstrap/bootstrap.py:48]: It's supposed to be more effective, as it only lazily creates the strings I think

Comment (oleorhagen) [reply src/mender/bootstrap/bootstrap.py:48]: https://docs.python.org/3/howto/logging.html#optimization

But, I'm a bit ambivalent here, kinda like the f-strings more

File: src/mender/_version.py (hunk 2 above)
Comment (lluiscampos) [reply src/mender/_version.py:14]: II don't know what is the best practice neither. Let's keep master for now, I'll look a bit on other packages

Comment (olehermanse) [reply src/mender/_version.py:14]: `master` makes more sense to me. Version information is already in git, not hardcoding it into the source code makes things a lot easier:

https://www.youtube.com/watch?v=U-aIPTS580s

Comment (vpodzime) [reply src/mender/_version.py:14]: Yes, but it also make sense to have separate stable branches for backports.

**Review by vpodzime — APPROVED** [review]
ACK

**Pull request merged**